.circleci/config.yml

```diff
@@ -117,15 +117,15 @@ jobs:
       - run:
           name: Apply all migrations
           command: make db-migrate
-          no_output_timeout: 30m
```

Comment:
This becomes become too long, we should just upload a new version with the existing migration already applied. What we want to test is our new migrations anyway. 
(out of scope of this PR)

Reply:
There was a bug which made the migration so long. Now it is fixed.